File: bench/c/sum.c
#include <stdio.h>

long long sum(int n) {
    long long s = 0;
    for (int i = 1; i <= n; i++)
        s += i;
    return s;
}

int main(void) {
    printf("%lld\n", sum(1000000));
    return 0;
}

File: bench/zig/sum.zig
const std = @import("std");

export fn sum(n: i32) i64 {
    var s: i64 = 0;
    var i: i32 = 1;
    while (i <= n) : (i += 1)
        s += i;
    return s;
}

pub fn main() void {
    std.debug.print("{d}\n", .{sum(1000000)});
}